zowex/Cargo.toml

```diff
@@ -19,9 +19,13 @@ serde_json = "1.0.94"
 simple-error = "0.2.3"
 supports-color = "2.0.0"
 sysinfo = "0.28.2"
+tokio = { version = "1", features = ["io-util", "macros", "net", "rt-multi-thread"] }
```

Comment:
Can you specify the full version so we can lock it down?

Reply:
yep! fixed in incoming commit